packages/mobile/src/verify/VerificationFailedModal.tsx

```diff
@@ -29,56 +35,54 @@ export function VerificationFailedModal(props: Props) {
     navigateHome()
   }, [props.cancelVerification])
 
+  const onRetry = React.useCallback(() => {
+    props.toggleFornoMode(true)
+    props.setRetryVerificationWithForno(false) // Only prompt retry with forno once
+    setIsDismissed(true)
+    navigate(Screens.VerificationEducationScreen)
```

Comment:
Renavigate uesrs to `VerificationEducationScreen` when they retry, so they will have to click start again. Lmk if anyone has thoughts on instead redirecting to `VerificationLoadingScreen`- this is more direct but was concerned users wouldn't know that verification had been started again

Reply:
I like sending users back to `VerificationEducationScreen`

Reply:
I'm fine with that too. Verification is a complex flow. Starting it again from the beginning seems safer.